src/types.ts

```diff
@@ -0,0 +1,20 @@
+export type NodeConfig = {
+  bootnodeEnodes?: string[],
+  networkID?: number,
+  maxPeers?: number,
+  genesis?: string,
+  nodeDir?: string,
+  keyStoreDir?: string,
+  enodes?: string,
+  noDiscovery?: boolean,
+  syncMode?: number
+  useLightweightKDF?: boolean,
+  ipcPath?: string
+  logFile?: string,
+  logFileLogLevel?: number,
+}
+
+export type Account = {
+  address: string,
```

Comment:
ditto
`account.account` is an index?
feels weird and unnecessary to me

Reply:
Yeah this is actually meant to mirror some geth internals and maybe it was a decision of the initial developers of the bridge that you should essentially pick an index when you select your `current account` and then all operations would happen against that account. So the flow on the consumer side would be something like:
```typescript
const geth = new RNGeth();
geth.setAccount(index);
geth.doSomething(); // would target current account
```

But since we transitioned to a model where we essentially do `geth.doSomething(account)` and not rely on the concept of a `currentAccount` we can and rightly should get rid of this.